wd_helper.au3

```diff
@@ -3478,19 +3478,22 @@ EndFunc   ;==>__WD_Base64Decode
 
 ; #INTERNAL_USE_ONLY# ===========================================================================================================
 ; Name ..........: __WD_ErrHnd
-; Description ...: Dummy error handler
-; Syntax ........: __WD_ErrHnd()
-; Parameters ....: None
+; Description ...: COM Error handler
+; Syntax ........: __WD_ErrHnd($oError)
+; Parameters ....: $oError              - Error object.
 ; Return values .: None
 ; Author ........: mLipok
 ; Modified ......:
 ; Remarks .......:
-; Related .......:
+; Related .......: __WD_UpdateExtractor, _WD_DownloadAsBinary
 ; Link ..........:
 ; Example .......: No
 ; ===============================================================================================================================
-Func __WD_ErrHnd()
-
+Func __WD_ErrHnd($oError)
+	Local Const $sFuncName = "__WD_ErrHnd"
+	If $_WD_DEBUG = $_WD_DEBUG_None Then Return
+	__WD_Error($sFuncName, $oError.number, "err.windescription: " & $oError.windescription, $oError.scriptline)
+	__WD_Error($sFuncName, $oError.number, "err.description is: " & $oError.description, $oError.scriptline)
 EndFunc   ;==>__WD_ErrHnd
```

Comment:
Pretty sure every other instance of __WD_Error is used within the context of SetError(). Also, I don't see why you need two separate calls to __WD_Error here.

Reply:
> Pretty sure every other instance of __WD_Error is used within the context of SetError()

When you back from com error handler functuon, there is no need to set @error or @extended

It will be set automatically by internal AutoIt rules to $oError.number

Reply:
> Also, I don't see why you need two separate calls to __WD_Error here.

I can change it when I be on my PC .... currently usin my smart phone.

Reply:
Thanks for the explanation @mlipok 👍 .

Reply:
> > Also, I don't see why you need two separate calls to __WD_Error here.
> 
> I can change it when I be on my PC .... currently using my smart phone.

Please take a look on: [simplify __WD_Error usage](https://github.com/Danp2/au3WebDriver/pull/513/commits/d8eb01b4027870096132351b7b3cd7ebe173bdc6)

Reply:
should we consider this as resolved ?